boards/forms.py

```diff
@@ -20,5 +20,9 @@ def update(self, *args, **kwargs):
 		update_board.save()
 		return update_board
 
+	def archive(self, *args, **kwargs):
```

Comment:
better to remove *args, **kwargs then then just pass the argument.
e.g 
`def archive(self, board):`